docs/setup/tomcat.md

```diff
@@ -57,6 +57,11 @@ As the SSL certificate is absolutely required, at least for the CAS module, you
 keytool -import -alias cert_ssl -file /var/www/georchestra/ssl/georchestra.crt -keystore /etc/tomcat8/keystore
 ```
 
+Or, if you have a Let's Encrypt certificate using Certbot :
+```
+keytool -import -alias cert_ssl -file /etc/letsencrypt/live/[your_sdi_domain_name]/cert.pem -keystore /etc/tomcat8/keystore
```

Comment:
Thanks. This duplicates a lot of information.
I'd rather have a comment on the above command to specify the certificate path in the "Let's Encrypt" case